src/popup/router/pages/ImportAccount.vue

```diff
@@ -36,6 +36,7 @@ export default {
     async importAccount() {
       if (this.mnemonic) {
         this.loading = true;
+        this.mnemonic = this.mnemonic.trim();
```

Comment:
In the Base app, we have a bit more complex formatting: https://github.com/aeternity/aepp-base/blob/3fb6a7c3264aab8ad22c1ae65685910858951e9b/src/components/AeInputMnemonic.vue#L23-L27
It handles additional cases:
- the first capital letter (actual for mobile)
- multiple spaces between words
- removing of not allowed symbols